R/loo_moment_matching.R

```diff
@@ -286,14 +286,20 @@ loo_moment_match_i <- function(i,
     # 1. match means
     trans <- shift(x, uparsi, lwi)
     # gather updated quantities
-    quantities_i <- update_quantities_i(x, trans$upars,  i = i,
+    quantities_i <- tryCatch(update_quantities_i(x, trans$upars,  i = i,
```

Comment:
```suggestion
    quantities_i <- try(update_quantities_i(x, trans$upars,  i = i,
```

Here you were using `tryCatch` but in the other cases below you use `try`. I would just use `try` everywhere for consistency (it's also simpler to use than `tryCatch`).